style.css

```diff
@@ -11,6 +11,10 @@ body {
     padding: 0;
 }
 
+body::-webkit-scrollbar {
+display: none;
```

Comment:
the vertical bar is still there on my end , it is very small vertical bar but it is there . i apperichate your work. also i  maybe going for horizontal card on computer screens as i found a fix that i moved all my style.css of body to '#root' expect 'margin 0 and padding 0' so its kinda worked , but it collapse my whole project and i have to do it from scratch